.github/workflows/push.yaml

```diff
@@ -0,0 +1,206 @@
+name: Push
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  DOCKER_FILE_PATH: build/Dockerfile
+  GOLANG_VERSION: 1.15.2
+  OPERATOR_SDK_VERSION: "0.15.2"
```

Comment:
`OPERATOR_SDK_VERSION` is not required here since this is a controller and doesn't use `operator-sdk`